webclient/src/views/view/view-view.js

```diff
@@ -519,6 +524,22 @@ navigatum.registerView("view", {
         );
       }
     },
+    loadModalRoomfinderMap: function () {
+      const map = this.view_data.maps.roomfinder.available[mapIndex];
```

Comment:
```suggestion
      const map = this.view_data.maps.roomfinder.available[this.state.map.roomfinder.selected_index];
```
This was a mistake. Now the function doesn't throw an error anymore at least, but the cross is still not at the right place. I think the reason for that is that the modal has always size 0x0 if it is not visible. And when this function is called, it is not yet visible (or the browser hasn't re-layouted yet). I'm not sure what the best solution here is. The easiest is to do a delay or polling until the modal is visible (or the size non-zero).